File: branches/Medusa/Source/Fantasy.Jobs/Scheduling/UncPath.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Runtime.Serialization;
using Fantasy.XSerialization;

namespace Fantasy.Jobs.Scheduling
{
    [DataContract]
    [XSerializable("uncPath", NamespaceUri = Consts.ScheduleNamespaceURI)]
    public class UncPath
    {
        [DataMember]
        [XAttribute("path")]
        public string Path { get; set; }

        [DataMember]
        [XAttribute("username")]
        public string UserName { get; set; }

        [DataMember]
        [XAttribute("password")]
        public string Password { get; set; }
    }
}

File: AutoConverted/Fantasy.Jobs/Scheduling/UncPath.java
package Fantasy.Jobs.Scheduling;

import Fantasy.XSerialization.*;

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
//[DataContract, XSerializable("uncPath", NamespaceUri = Consts.ScheduleNamespaceURI)]
public class UncPath
{
//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
	//[DataMember, XAttribute("path")]
	private String privatePath;
	public final String getPath()
	{
		return privatePath;
	}
	public final void setPath(String value)
	{
		privatePath = value;
	}

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
	//[DataMember, XAttribute("username")]
	private String privateUserName;
	public final String getUserName()
	{
		return privateUserName;
	}
	public final void setUserName(String value)
	{
		privateUserName = value;
	}

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
	//[DataMember, XAttribute("password")]
	private String privatePassword;
	public final String getPassword()
	{
		return privatePassword;
	}
	public final void setPassword(String value)
	{
		privatePassword = value;
	}
}